### PR TITLE
Added LED_RGBW_Cree-PLCC-8_5,0x5,2mm Footprint

### PR DESCRIPTION
**Pull request by fabikroeger**
Added LED_RGBW_Cree-PLCC-8_5,0x5,2mm Footprint
Datasheet URL: https://www.cree.com/led-components/media/documents/ds-CLQ6A-TKW.pdf

Screenshot:
LED_RGBW_Cree_PLCC_8_Footprint

KLC helperscript: 
F5.2 violation: For me the fabric layer is okay - no reason for me why the script is upset.
F6.2 violation: Also unclear why that script is upset.
F9.3 violation: No proper 3D model found

---

All contributions to the kicad library must follow the [KiCad library convention](http://kicad-pcb.org/libraries/klc/)

Thanks for creating a pull request to contribute to the KiCad libraries! To speed up integration of your PR, please check the following items:

- [x] Provide a URL to a datasheet for the footprint(s) you are contributing
- [x] An example screenshot image is very helpful 
- [ ] If there are matching symbol or 3D model pull requests, provide link(s) as appropriate
- [ ] Check the output of the Travis automated check scripts - fix any errors as required
- [x] Give a reason behind any intentional library convention rule violation.

### CODE REVIEW

**Comment by chschlue** [comment]
Welcome and thanks for your first contribution.

All CI errors appear to be valid at first glance, please fix.
Also, you need to commit using an email address linked to your GH account.

**Comment by chschlue** [comment]
- [ ] Include the 3D model even if it doesn't exist. That way it can be added later without touching the footprint again.
- [ ] use dots as decimal separators in the description
- [ ] you can use custom pad shapes with fitting geometric primitives instead of graphic polygons on F.Copper

**Comment by fabikroeger** [comment]
> * [x]  Include the 3D model even if it doesn't exist. That way it can be added later without touching the footprint again.
> * [x]  use dots as decimal separators in the description
> * [x]  you can use custom pad shapes with fitting geometric primitives instead of graphic polygons on F.Copper

**Comment by chschlue** [comment]
cree-rgbw

Please fix dimensions.

**Comment by fabikroeger** [comment]
thx for the review! I will do that.

**Comment by fabikroeger** [comment]
I will take more patience for the future, no problem :-)

**Comment by poeschlr** [comment]
If you want to speed up the process then help us by providing a dimensioned drawing like the one @chschlue  made. A tutorial for it is found here: https://forum.kicad.info/t/how-to-check-footprint-correctness/9279

**Comment by chschlue** [comment]
@fabikroeger are you still on this?

**Comment by fabikroeger** [comment]
Hey @chschlue, 
thanks for asking! Yes i am.  I was three weeks out of my country and now busy at work. So i will get back on that soon.

**Comment by fabikroeger** [comment]
Now i am back @chschlue =)

**Comment by evanshultz** [comment]
The silk lines to mark pin 1 don't look even from the silk outline. And FYI, IPC has been been showing silk removal to mark pin 1. I have experienced this to be better because extra silk is often removed due to pads in close proximity to a footprint and then the silk mark is removed. So cutting out silk around the pin 1 corner may be a better way to go than adding silk near pin 1.

**Comment by fabikroeger** [comment]
Thx for the comment! I had a look at an other SMD RGB LED and decided to go with that one.

**Comment by chschlue** [comment]
While this isn't exactly "silk removal to mark pin 1", it _is_ the most common style in LED_SMD.

But why did you remove the custom pads again?

**Comment by fabikroeger** [comment]
> But why did you remove the custom pads again?
That was not to be intended! now they are back.

Also the silk removing should be as asked =) 

Thanks for the patience!

File: LED_SMD.pretty/LED_RGBW_Cree-PLCC-8_5.0x5.2mm_P1.1mm.kicad_mod [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+         (xy -0.85 0.285) (xy 0.9 0.285) (xy 0.9 -0.355) (xy 0 -0.355) (xy -0.85 -0.565)
+) (width 0))
+    ))
+  (pad 3 smd custom (at -1.8 0.435) (size 0.1 0.1) (layers F.Cu F.Paste F.Mask)
+    (zone_connect 0)
+    (options (clearance outline) (anchor circle))
+    (primitives
+      (gr_poly (pts
+         (xy -0.85 -0.285) (xy -0.85 0.565) (xy 0 0.355) (xy 0.9 0.355) (xy 0.9 -0.285)
+         (xy 0.9 0.355) (xy 0.9 -0.285)) (width 0))
```
Comment (chschlue) [inline LED_SMD.pretty/LED_RGBW_Cree-PLCC-8_5.0x5.2mm_P1.1mm.kicad_mod]: Two redundant points

File: LED_SMD.pretty/LED_RGBW_Cree-PLCC-8_5.0x5.2mm_P1.1mm.kicad_mod [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    ))
+  (pad 7 smd custom (at 1.8 -0.435) (size 0.1 0.1) (layers F.Cu F.Paste F.Mask)
+    (zone_connect 0)
+    (options (clearance outline) (anchor circle))
+    (primitives
+      (gr_poly (pts
+         (xy 0.85 0.305) (xy 0.85 -0.565) (xy -0.83 -0.565) (xy -0.83 -0.415) (xy -2.4 -0.415)
+         (xy -2.4 0.305) (xy -2.4 -0.415) (xy -2.4 0.305)) (width 0))
```
Comment (chschlue) [inline LED_SMD.pretty/LED_RGBW_Cree-PLCC-8_5.0x5.2mm_P1.1mm.kicad_mod]: Two redundant points

File: LED_SMD.pretty/LED_RGBW_Cree-PLCC-8_5.0x5.2mm_P1.1mm.kicad_mod [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    ))
+  (pad 6 smd custom (at 1.8 0.435) (size 0.1 0.1) (layers F.Cu F.Paste F.Mask)
+    (zone_connect 0)
+    (options (clearance outline) (anchor circle))
+    (primitives
+      (gr_poly (pts
+         (xy 0.85 -0.305) (xy 0.85 0.565) (xy -0.83 0.565) (xy -0.83 0.415) (xy -2.4 0.415)
+         (xy -2.4 -0.305) (xy -2.4 0.415) (xy -2.4 -0.305)) (width 0))
```
Comment (chschlue) [inline LED_SMD.pretty/LED_RGBW_Cree-PLCC-8_5.0x5.2mm_P1.1mm.kicad_mod]: Two redundant points

**Pull request closed without merging**